src/components/ImportStatus.js

```diff
@@ -44,11 +44,19 @@ const ImportStatusItem = ({ job: { targetWorkspace, jobId }, onDone }) => {
     const response = await fetchImportStatus()
     const { message, status } = response
 
-    if (!_.includes(status, ['PENDING', 'RUNNING'])) {
+    // avro-import statuses: PENDING, RUNNING, SUCCESS, ERROR
+    // import service statuses: Pending, Translating, ReadyForUpsert, Upserting, Done, Error
+    // TODO: only need to support both sets of statuses during the transition from avro-import to import service.
+    // once import servie is fully adopted, we can/should remove the avro-import status values.
+
+    if (!_.includes(status, ['PENDING', 'RUNNING', 'Pending', 'Translating', 'ReadyForUpsert', 'Upserting'])) {
       Utils.switchCase(status,
         ['SUCCESS', () => notify('success', 'Data imported successfully.',
           { message: `Data import to workspace "${namespace} / ${name}" is complete. Please refresh the Data view.` })],
+        ['Done', () => notify('success', 'Data imported successfully.',
```

Comment:
Could we merge these in with the above. ('Done' || 'SUCCESS')? If not, could we pull the lambda out into a variable so we can just reuse it? Same thing with error.

Reply:
does that syntax even work?
```
Utils.switchCase(status,
        [('SUCCESS' || 'Done'), () => ...
```
I tried it and got a JS error, but possibly did something wrong. I did think about pulling the lambda out into a var, but didn't think it was worth it, since this code should only exist for a couple weeks and then return to a non-duplicated case. If you think it's worth it, I'd do it though!

Reply:
I don't think that syntax works. I wasn't sure what the right syntax is. Looks like the method doesn't support boolean operations like that.

I would say we should pull out the code into a variable, mainly because it isn't too difficult and I think string duplication is pretty rough to have in code (even if only for a brief period of time)

Reply:
@blrubenstein done!

Reply:
FYI, If you wanted a more flexible expression in the test, you can use `Utils.cond`, at the expense of having to repeat the thing you're checking:
```
Utils.cond(
  [status === 'SUCCESS' || status === 'Done', () => ...],
  [_.lowerCase(status) === 'error', () => ...],
  () => ... // default case
)
```
(something like that... I didn't test the code above)